cpp/cmake/modules/ConfigureCUDA.cmake

```diff
@@ -25,8 +25,8 @@ endif()
 list(APPEND CUML_CUDA_FLAGS --expt-extended-lambda --expt-relaxed-constexpr)
 
 # set warnings as errors
-# list(APPEND CUML_CUDA_FLAGS -Werror=cross-execution-space-call)
-# list(APPEND CUML_CUDA_FLAGS -Xcompiler=-Wall,-Werror,-Wno-error=deprecated-declarations)
+list(APPEND CUML_CUDA_FLAGS -Werror=cross-execution-space-call, -Wno-error=unknown-pragmas)
```

Comment:
-Wno-error=unknown-pragmas  

This is just because there is one "#pragma unroll ignored" warning from nvcc that I couldn't figure out.

Reply:
As in the compiler is producing a diagnostic that  "#pragma unroll" is being ignored? I am not aware of a better approach to suppressing those.

Reply:
Yes, it's weird, because #pragma unroll in other .cu files works fine.  Let me find the file...

Reply:
It's infer.cu: 

```
[26/227] Building CUDA object CMakeFiles/cuml++.dir/src/fil/infer.cu.o
/home/mharris/rapids/cuml/cpp/src/fil/infer.cu:47: warning: ignoring #pragma unroll  [-Wunknown-pragmas]
   47 |     for (int i = 0; i < NITEMS; i++)
      | 
/home/mharris/rapids/cuml/cpp/src/fil/infer.cu:65: warning: ignoring #pragma unroll  [-Wunknown-pragmas]
   65 |     for (int i = 0; i < N; ++i)
      | 
/home/mharris/rapids/cuml/cpp/src/fil/infer.cu:116: warning: ignoring #pragma unroll  [-Wunknown-pragmas]
  116 |     for (int i = 0; i < NITEMS; i++)
      | 
```

Reply:
Ah, it's host compiler warnings due to use in a host/device function. Fixed.